### PR TITLE
WICKET-6672: restore use of wicket Duration for Ajax timer

### PR DESCRIPTION
**Pull request by reiern70**

### CODE REVIEW

**Comment by reiern70** [comment]
It seems some files have LR and other CRLF.

**Comment by reiern70** [comment]
@solomax 

Thanks.

File: wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java [hunk 1]
```diff
@@ -52,9 +52,22 @@
 
 	/**
 	 * Construct.
-	 * 
+	 *
+	 * @param updateInterval
+	 *            {@link org.apache.wicket.util.time.Duration} between AJAX callbacks
+	 *
+	 * @deprecated Since Wicket 9 this class is obsolete and no more used. It will be removed in Wicket 10. Use {@link java.time.Duration} instead
```
Comment (martin-g) [inline wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java]: s/this class/this constructor/

Comment (martin-g) [reply wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java]: You should use `{@link AbstractAjaxTimerBehavior(Duration)}` instead, i.e. point to the new constructor.

File: wicket-core/src/main/java/org/apache/wicket/ajax/AjaxSelfUpdatingTimerBehavior.java [hunk 2]
```diff
@@ -33,11 +33,24 @@
 {
 	private static final long serialVersionUID = 1L;
 
+	/**
+	 * Construct.
+	 *
+	 * @param updateInterval
+	 *            {@link org.apache.wicket.util.time.Duration} between AJAX callbacks
+	 *
+	 * @deprecated Since Wicket 9 this class is obsolete and no more used. It will be removed in Wicket 10. Use {@link java.time.Duration} instead
```
Comment (martin-g) [inline wicket-core/src/main/java/org/apache/wicket/ajax/AjaxSelfUpdatingTimerBehavior.java]: see above

File: wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java (hunk 1 above)
Comment (reiern70) [reply wicket-core/src/main/java/org/apache/wicket/ajax/AbstractAjaxTimerBehavior.java]: Done!

File: wicket-core/src/main/java/org/apache/wicket/ajax/AjaxSelfUpdatingTimerBehavior.java (hunk 2 above)
Comment (reiern70) [reply wicket-core/src/main/java/org/apache/wicket/ajax/AjaxSelfUpdatingTimerBehavior.java]: Done!

**Comment by reiern70** [comment]
@martin-g Thanks for comments. Fixed them

**Pull request merged**